api/v3/openapi.yaml

```diff
@@ -1011,15 +1011,27 @@ components:
       type: object
       required:
         - topic_name
-        - partitions_count
-        - replication_factor
       properties:
         topic_name:
           type: string
         partitions_count:
           type: integer
         replication_factor:
           type: integer
+        replicas_assignments:
+          type: array
+          items:
+            type: object
+            required:
+              - partition
+              - replicas
+            properties:
+              partition:
+                type: integer
+              replicas:
```

Comment:
Same here. Since these won't be links to the replicas, then you should use the identifier name instead. For replicas, we use `broker_id`, so this here should be `broker_ids`.